src/pseudopeople/interface.py

```diff
@@ -714,7 +720,7 @@ def generate_women_infants_and_children(
         filters.append(DataFilter(DATASET_SCHEMAS.wic.date_column_name, "==", year))
         seed = seed * 10_000 + year
     if state is not None:
-        state_column = DATASET_SCHEMAS.wic.state_column_name
+        state_column = cast(str, DATASET_SCHEMAS.wic.state_column_name)
```

Comment:
It's unclear to me when you need to cast versus `state_column: str = DATASET_SCHEMAS.wic.state_column_name`

Reply:
same here!